app/controllers/sorry_controller.rb

```diff
@@ -0,0 +1,23 @@
+# Page to display if something goes wrong with the request
+# e.g. redirect from ILLiad logon or ScanAndDeliver pages
+class SorryController < ApplicationController
+  def unable
+    @please_contact = please_contact
+    render status: :forbidden
```

Comment:
Why `forbidden/403`? Do we know for sure that is the reasoning for routing to this page?